### PR TITLE
1990e add va file number

### PR DESCRIPTION
**Pull request by annekainicUSDS**
Adding VA file number to the 1990e from feedback we received during testing with Shay: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1657. Add it to the schema so I can make the change on the front-end.

### CODE REVIEW

**Comment by lihanli** [comment]
can you update the version in package.json?

File: src/schemas/transfer-benefits/schema.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ let schema = {
   type: 'object',
   additionalProperties: false,
   definitions: _.pick(definitions,
-    'privacyAgreementAccepted',
+    'privacyAgreementAccepted', [
```
Comment (lihanli) [inline src/schemas/transfer-benefits/schema.js]: not needed i think?

Comment (annekainicUSDS) [inline src/schemas/transfer-benefits/schema.js]: You're right, I don't think they're necessary, I was just copying from change-of-program, which has them. I can remove.

**Comment by annekainicUSDS** [comment]
@lihanli updates made!

**Comment by jbalboni** [comment]
FYI @raquelromano-gov, I think you have an item related to this.

File: dist/transfer-benefits-schema.json [hunk 2]
```diff
@@ -458,8 +458,24 @@
...
   },
+  "anyOf": [
+    {
+      "required": [
+        "vaFileNumber"
+      ]
+    },
+    {
+      "required": [
+        "relativeSocialSecurityNumber"
```
Comment (raquelromano-gov) [inline dist/transfer-benefits-schema.json]: @annekainicUSDS @lihanli 
Just noticed this--shouldn't this be veteranSocialSecurityNumber?

I believe the relative is the applicant (relative of the veteran), but the veteran is the one who can offer a va file # in lieu of a social security number.

**Pull request merged**